app/src/main/res/values-ja/strings.xml

```diff
@@ -39,4 +39,7 @@
     <string name="session_feedback">セッションフィードバック</string>
     <string name="info_sponsors_title">スポンサー</string>
     <string name="info_contributors_title">コントリビュータ</string>
+    <!-- Notification -->
+    <string name="notitication_title">%1$s がもうすぐ始まります。</string>
+    <string name="notification_message">%1$s ~ %2$s @ %3$s</string>
```

Comment:
@konifar Tried replying through email but I guess my comment got lost. Yeah, we need to add them on oneskyapp and import it later. 👍  You can leave it as is and later we add to onesky.